### PR TITLE
DDCE-1793: updated to govuk aswell, fixed various bugs and tidied up views

### PR DESCRIPTION
**Pull request by fraser-sprott**

### CODE REVIEW

File: app/connectors/HmrcTierConnector.scala [hunk 1]
```diff
@@ -90,8 +90,8 @@ class HmrcTierConnector @Inject()(client: HttpClient, configuration: Configurati
     val optMapped = Map(HeaderTags.ETAG -> etagFromSession, HeaderTags.X_TXID -> xtxidFromSession)
 
     Logger.info(
```
Comment (anburoach) [inline app/connectors/HmrcTierConnector.scala:92]: plz fix the Logger deprecated warnings by either extends Logging or Logger(this.getClass)

File: app/assets/stylesheets/application.scss [hunk 2]
```diff
@@ -0,0 +1,8 @@
+$govuk-assets-path: "/play-mtp-frontend/assets/lib/govuk-frontend/govuk/assets/";
+
+@import "lib/govuk-frontend/govuk/all";
+
+.app-reference-number {
+  display: block;
+  font-weight: bold;
+}
```
Comment (anburoach) [inline app/assets/stylesheets/application.scss:8]: new line plz

File: app/views/Enrol.scala.html [hunk 3]
```diff
@@ -14,36 +14,32 @@
...
-    <p>@messages("Service.notEnrolled.text3")</p>
+    <p class="govuk-body">@messages("Service.notEnrolled.text3")</p>
 
-    <p>@messages("Service.notEnrolled.text4")</p>
+    <p class="govuk-body">@messages("Service.notEnrolled.text4")</p>
 
-    <p>
-        <a href='https://www.gov.uk/paye-online/enrol' class="button" role="button">@messages("Service.notEnrolled.button")</a>
-    </p>
+    <a class="govuk-button" href='https://www.gov.uk/paye-online/enrol' class="button" role="button">@messages("Service.notEnrolled.button")</a>
```
Comment (anburoach) [inline app/views/Enrol.scala.html:44]: class comes 2 times.. plz remove 2nd one

File: app/views/ErrorPage.scala.html [hunk 4]
```diff
@@ -15,90 +15,85 @@
...
+    }
 
-        @if(code == 63088 || code == 64989 || code == 10003) {
-            <p>
-                @Html(messages("ServiceMessage." + (code) + ".2"))
-            </p>
+    <div class="govuk-form-group">
+        @code match {
+            case 63082 => {<a class="govuk-link" id="link-try-again" href='javascript: window.history.go(-1)'>@messages("Service.tryagain")</a>}
+            case 63090 => {<a class="govuk-link" id="link-add-another" href='javascript: window.history.go(-1)'>@messages("Service.excludeanotheremployee")</a>}
```
Comment (anburoach) [inline app/views/ErrorPage.scala.html:73]: no differences on lines 73 and 74..so plz make this into one line using |

File: app/views/GovukLayoutWrapper.scala.html [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+        Some(UserResearchBanner(url = config.urBannerLink))
+    } else {
+        None
+    }
+}
+
+@beforeContentBlock = {
+    <div>
+        <div class="hmrc-banner govuk-!-display-inline-block">
+            <div class="hmrc-organisation-logo">
```
Comment (anburoach) [inline app/views/GovukLayoutWrapper.scala.html:76]: I think no need to display hmrc logo... could you please check with design team on this?

Comment (fraser-sprott) [reply app/views/GovukLayoutWrapper.scala.html:76]: Done the demo to the design team who are happy with the change and still within the current prototype, believe this is outside of scope for this

File: app/views/ErrorPage.scala.html (hunk 4 above)
Comment (fraser-sprott) [reply app/views/ErrorPage.scala.html:73]: good point

File: app/views/exclusion/NinoExclusionSearchForm.scala.html [hunk 6]
```diff
@@ -14,113 +14,96 @@
...
+            @if(searchForm("status").hasErrors) {
+                @if(alreadyExists) {
+                    @Html(messages("ExclusionSearch.Fail.Exists.Desc", messages("BenefitInKind.label." + iabdType), excludedEmployeesLink, "</a>"))
+                } else {
+                    @Html(messages("ExclusionSearch.Fail.Desc"))
                 }
+            }
+            <ul class="govuk-list govuk-error-summary__list">
+                <li>
+                    @if(searchForm("status").hasErrors){
```
Comment (anburoach) [inline app/views/exclusion/NinoExclusionSearchForm.scala.html:60]: we can use govukErrorSummary to dsiplay the error messages

File: app/views/exclusion/NoNinoExclusionSearchForm.scala.html [hunk 7]
```diff
@@ -14,107 +14,83 @@
...
-                    <li><a href="#surnameContainer">@searchForm.error("surname").get.message.dropRight(1)</a></li>
-                }
-                @if(searchForm("dob").hasErrors) {
+            }
 
-                    @if(searchForm.error("dob").get.message.equals("error.empty.dob")){
-                        <li><a href="#dob">@messages("error.empty.dob").dropRight(1)</a></li>
+            <div class="govuk-error-summary__body">
+                <ul class="govuk-list govuk-error-summary__list">
+                    @if(searchForm("firstname").hasErrors) {
```
Comment (anburoach) [inline app/views/exclusion/NoNinoExclusionSearchForm.scala.html:63]: we can use govukErrorSummary to dsiplay the error messages...

File: project/AppDependencies.scala [hunk 8]
```diff
@@ -9,7 +9,8 @@ object AppDependencies {
     "uk.gov.hmrc" %% "bootstrap-frontend-play-27" % "4.2.0",
     "uk.gov.hmrc" %% "play-ui" % "8.21.0-play-27",
```
Comment (anburoach) [inline project/AppDependencies.scala:10]: remove play-ui and govuk-template libraries

**Review by anburoach — APPROVED** [review]
Approved

**Pull request merged**